generic/BRGenericRipple.c

```diff
@@ -136,10 +136,10 @@ genericRippleWalletGetBalance (BRGenericWallet wallet) {
     return createUInt256(rippleWalletGetBalance(ripple));
 }
 
-static BRGenericTransfer
-genericRippleWalletManagerRecoverTransfer (uint8_t *bytes,
-                                           size_t   bytesCount) {
-    return rippleTransactionCreateFromBytes (bytes, (int) bytesCount);
+static BRArrayOf(BRGenericTransfer)
+genericRippleWalletManagerRecoverTransfersFromRawTransaction (uint8_t *bytes,
+                                                            size_t   bytesCount) {
+    return NULL;
```

Comment:
Reasonable.  Ripple will not event attempt to extract transfers from `bytes`.  Just return NULL here.